build_runner/lib/src/package_graph/apply_builders.dart

```diff
@@ -289,3 +291,19 @@ Map<String, List<BuilderApplication>> _applyWith(
   }
   return applyWith;
 }
+
+/// Runs [fn] in an error handling [Zone].
+///
+/// Any calls to [print] will be logged with `log.info`, and any errors will be
+/// logged with `log.severe`.
```

Comment:
I think we need to find a way to bubble up the error and cancel the overall build. Maybe it will be enough to do a null check later on?

Reply:
Checking for null was enough. Added a corresponding test.